Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/http/server.py", line 1314, in <module>
    test(
  File "/usr/local/lib/python3.12/http/server.py", line 1261, in test
    with ServerClass(addr, HandlerClass) as httpd:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/socketserver.py", line 457, in __init__
    self.server_bind()
  File "/usr/local/lib/python3.12/http/server.py", line 1308, in server_bind
    return super().server_bind()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/local/lib/python3.12/socketserver.py", line 478, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
10.12.27.110 - - [20/Sep/2025 08:36:26] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:26] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:26] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:27] "GET /%EC%B2%A0%EC%82%B0%20%EB%A7%A5%EB%8F%84%EB%82%A0%EB%93%9C.png HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:30] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:36:30] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:36:42] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:52] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:53] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:53] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:53] "GET /%EC%B2%A0%EC%82%B0%20%EB%A7%A5%EB%8F%84%EB%82%A0%EB%93%9C.png HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:36:55] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:36:55] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:39:41] "GET / HTTP/1.1" 304 -
10.12.27.110 - - [20/Sep/2025 08:39:41] "GET /script.js HTTP/1.1" 304 -
10.12.27.110 - - [20/Sep/2025 08:39:41] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:39:42] "GET /%EC%B2%A0%EC%82%B0%20%EB%A7%A5%EB%8F%84%EB%82%A0%EB%93%9C.png HTTP/1.1" 304 -
10.12.27.110 - - [20/Sep/2025 08:40:03] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:40:03] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:40:03] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:40:04] "GET /%EC%B2%A0%EC%82%B0%20%EB%A7%A5%EB%8F%84%EB%82%A0%EB%93%9C.png HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:40:05] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:40:05] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:47:54] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:47:54] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:47:54] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:47:55] "GET /%EC%B2%AD%EB%85%84%EB%8F%99%20%EB%A1%9C%EB%B9%84.jpg HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:47:55] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:47:55] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:48:00] "GET /%EC%B2%A0%EC%82%B0%20%EB%A7%A5%EB%8F%84%EB%82%A0%EB%93%9C.png HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:03] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:04] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:04] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:04] "GET /%EC%B2%AD%EB%85%84%EB%8F%99%20%EB%A1%9C%EB%B9%84.jpg HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:05] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:48:05] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:48:09] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:09] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:09] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:10] "GET /%EC%B2%AD%EB%85%84%EB%8F%99%20%EB%A1%9C%EB%B9%84.jpg HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:48:11] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:48:11] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:51:29] "GET / HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:51:29] "GET /style.css HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:51:29] "GET /script.js HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:51:30] "GET /%EC%B2%AD%EB%85%84%EB%8F%99%20%EB%A1%9C%EB%B9%84.jpg HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:51:30] code 404, message File not found
10.12.27.110 - - [20/Sep/2025 08:51:30] "GET /favicon.ico HTTP/1.1" 404 -
10.12.27.110 - - [20/Sep/2025 08:51:32] "GET /%EC%B2%AD%EB%85%84%EB%8F%99%20%ED%9A%8C%EC%9D%98%EC%8B%A4%202%EB%B2%88.jpg HTTP/1.1" 200 -
10.12.27.110 - - [20/Sep/2025 08:51:37] "GET /%EC%B2%A0%EC%82%B0%20%EB%A7%A5%EB%8F%84%EB%82%A0%EB%93%9C.png HTTP/1.1" 200 -
